tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml

```diff
@@ -15,22 +15,26 @@ tests:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
+      - nrf54l15pdk_nrf54l15_cpuapp
```

Comment:
`nrf54l15pdk_nrf54l15_cpuapp_ns` support was introduced recently. Can we add it for the tests?

Reply:
Discussed offline, we will do it in the other ticket: https://nordicsemi.atlassian.net/browse/NCSDK-26892